### DISCUSSION TITLE
remote-config versioning issues: v10.0.0 vs 10.0.1?

### DISCUSSION THREAD

**Discussion by draperunner** [invertase/react-native-firebase#4590] [General]
Hi! I'm upgrading all my firebase dependencies to v10, but it seems remote-config already was on v10.0.1 before it was "upgraded" to v10.0.0.

```npm info @react-native-firebase/remote-config```:

<img width="432" alt="image" src="https://user-images.githubusercontent.com/4339443/99913625-efb41480-2cf8-11eb-89c9-d99f31c2d348.png">

It also seems like both v10.0.0 (next) and v10.0.1 (latest) depend on `@react-native-firebase/app` *v9*, which is a problem.

**Comment by mikehardy** [comment]
The v9 dep is a big deal, I'll sort that. The v10.0.1/v10.0.0 was an artifact of me attempting to dry-run the big change and having it publish one anyway, but I did not think it would have any consequence other than more gray hairs for me.
I'm hoping to do a patch release shortly this week and it will mop up the v9 thing which I was unaware of - thanks for reporting it

**Reply by mikehardy** [reply]
This is better tracked with the issue someone logged here (as it is not really a discussion, it's a real issue) https://github.com/invertase/react-native-firebase/issues/4607

**Comment by loubackrethink** [comment]
I had same problem @draperunner, but when I change the version of react-native-firebase/remote-config to 10.0.0 at least has been build in both platforms.

**Reply by mikehardy** [reply]
follow that #4607 above and you'll see when it's resolved - it is just down to some CI check fixing and release engineering on my part, hoping today but could be a couple days unfortunately. There should be nothing code-wise wrong with v10.anything though, they were all built from the same actual code bits

**Reply by loubackrethink** [reply]
Thanks Mike, I'll follow up this issue and I agree that is nothing about wrong code in v10 because I tested all firebase services using v10 in production as well.

**Reply by mikehardy** [reply]
Fixed in the latest release v10.1.0 and that was published full-auto so this should not recur though it was a bit messy getting to this new "fixed version for all packages" state combined with some underlying CI work. Thanks for bearing with us.
